src/components/kpiWidget/barChart.js

```diff
@@ -103,25 +103,33 @@ const getBarChartData = (state, ownProps) => {
     return [];
   }
   const alarmsByRule = state.kpiReducer.alarmsByRule || [];
-  const alarmsByRuleLastDuration =
-    state.kpiReducer.alarmsByRuleLastDuration || [];
-
+  const alarmsByRuleLastDuration = state.kpiReducer.alarmsByRuleLastDuration || [];
+  const rulesAndActions = state.ruleReducer.rulesAndActions
   const top5Latest = alarmsByRule.sort(sortByCount).slice(0, 5);
   const chartData = [];
   top5Latest.forEach(data => {
-    let lastDurationData;
+      let lastDurationData;
```

Comment:
indentation is off in this block, please check it, as everything should be 2 space tabs